lib/hets-agent/application.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+require 'config'
+require 'pathname'
+
+module HetsAgent
+  # The Application class encapsulates some basic properties.
+  class Application
+    ROOT = Pathname.new(File.expand_path('../../../', __FILE__)).freeze
+    ENVIRONMENT = (ENV['HETS_AGENT_ENV'] || 'development').freeze
+
+    class << self
+      def root
+        ROOT
+      end
+
+      def env
+        environment = ENVIRONMENT.dup
```

Comment:
Is this really supposed to happen on every call to `env`?

Reply:
Fixed in c60cc61.